templates/travisCI.st

```diff
@@ -1,6 +1,10 @@
 | gitPath gitCache |
 Transcript cr; show: 'travis---->travisCI.st'.
 
+"If e.g. your tests change code in the image dynamically, the image will require you to set 
+author initials which might fail these tests. So we set some dummy initials here to avoid this."
+Utilities setAuthorInitials: 'travisCI'.
+
```

Comment:
The problem with this change is that it is not platform independent ... perhaps it only works in Squeak these days?

I think that `MetacelloPlatform current authorName:` would be a better choice ...